README.md

```diff
@@ -52,192 +52,9 @@ Visit the [official documentation for Media Chrome](https://www.media-chrome.org
 
 #### <a href="https://media-chrome.mux.dev/examples/" target="_blank">See all of the repo examples in action.</a>
 
-## Audio Example
```

Comment:
Let's keep the simple audio example here too, please. This will be a pretty common landing point for users and they're pretty different audiences. I think we could do better upholding the audio side of media chrome in general.